packages/widgets/src/blocks/legacy-widget/edit/index.js

```diff
@@ -131,7 +131,8 @@ function NotEmpty( {
 		);
 	}
 
-	const mode = isNavigationMode || ! isSelected ? 'preview' : 'edit';
+	const mode =
```

Comment:
This feels like it needs a comment with context to explain _why_ `idBase` is a determining factor in selecting the "mode".